lab1_task1c.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE HTML>
 <html>
-
+<!--rev dev-->
 <head>
   <meta charset="utf-8">
   <title>Завдання 1.c</title>
```

Comment:
`</br>` tag is not valid.

Reply:
Do not overuse blank lines. You rarely need more than one between the logical blocks.
```html
        Photo</br>
        <input type="file" value="Вибрати файл"></br>
        Comments</br>
        <textarea cols="30" rows="5" name="textarea"></textarea></br>
        <input type="submit" value="save">




    </form>
```

Reply:
Better use input names that describe the input. Like instead of:
```html
        Male<input type="radio" name="radio" value="male" checked="checked">Female
        <input type="radio" name="radio" value="female"></br>

        Current Status</br>
        <input type="checkbox" name="checkB" value="Student">Student</br>
        <input type="checkbox" name="checkB" value="Employed">Employed</br>
        <input type="checkbox" name="checkB" value="Unemployed">Unemployed</br>
```

Better use:
```html
        Male<input type="radio" name="sex" value="male" checked="checked">
        Female<input type="radio" name="sex" value="female"><valid_br_tag>

        Current Status<valid_br_tag>
        <input type="checkbox" name="status" value="Student">Student<valid_br_tag>
        <input type="checkbox" name="status" value="Employed">Employed<valid_br_tag>
        <input type="checkbox" name="status" value="Unemployed">Unemployed<valid_br_tag>
```

Reply:
agree